python/foglamp/services/south/server.py

```diff
@@ -154,6 +154,7 @@ class Server(FoglampMicroservice):
         """Executes async type plugin
         """
         await Ingest.start(self._core_management_host, self._core_management_port)
+        self._plugin_handle['ingest'] = Ingest
```

Comment:
was not it needed before? when Ingest  was used in POLL?

Reply:
poll section already contains this line.